src/test/compile-fail/empty-struct-with-braces-1.rs

```diff
@@ -8,13 +8,10 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-// compile-flags: -Z parse-only
+// Empty struct defined with braces shouldn't add names into value namespace
```

Comment:
Great to have a test for it, but.

Isn't this a surprising gotcha? `struct S {}` and `struct S;` being noticably different?.